scripts/install_dev_tools

```diff
@@ -173,42 +198,53 @@ else
     VENV="${venv_dir}"
 fi
 
+# Get the list of CLI tools to be installed from comma separated output
+readarray -td, cli_tools_arr <<<"$cli_tools"
```

Comment:
Fixed, replaced with shell substitution, however to make it working I need to add shellcheck disable=SC2206 for one line otherwise it will behave as one string rather then array.